main.nf

```diff
@@ -146,8 +146,7 @@ def helpMessage() {
       --freebayes_p                   Specify ploidy of sample in FreeBayes. Default: ${params.freebayes_p}
       --pileupcaller_bedfile          Specify path to SNP panel in bed format for pileupCaller.
       --pileupcaller_snpfile          Specify path to SNP panel in EIGENSTRAT format for pileupCaller.
-      --pileupcaller_majority_call    Specify majority calling. Default: random haploid
-      --pileupcaller_random_diploid   Specify diploid calling. Default: random haploid
+      --pileupcaller_method           Specify calling method to use. Options: randomHaploid, randomDiploid, majorityCall. Default: randomHaploid
```

Comment:
```suggestion
      --pileupcaller_method           Specify calling method to use. Options: randomHaploid, randomDiploid, majorityCall. Default: ${params.pileupcaller_method}
```

This should take whatever is set to default in `nextflow.config`, so speeds up default changes with less erorrs.